crypto/mem_clr.c

```diff
@@ -60,6 +60,7 @@
 #include <string.h>
 #include <openssl/crypto.h>
 
+extern unsigned char cleanse_ctr;
```

Comment:
Ah sorry, I linked to the wrong build logs. The original error can be found at:
https://travis-ci.org/openssl/openssl/jobs/87030551#L970

> mem_clr.c:63:15: error: no previous extern declaration for non-static variable

The warning is only showed by clang-3.6 (not gcc-5).